addons/60fps/addon.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "60FPS project player mode",
-  "description": "Alt+Click the green flag to toggle 60FPS.",
+  "name": "Customizable project framerate",
```

Comment:
In general, addons that start with "customizable" don't change anything unless you change the settings. I thought it was worth mentioning.

Reply:
My idea was this:
```suggestion
  "name": "Higher project framerate",
```

Reply:
> In general, addons that start with "customizable" don't change anything unless you change the settings. I thought it was worth mentioning.

It's like halfway, because the settings aren't the Scratch default, but they're not applied until the user applies them by Alt+clicking on the green flag. That puts the addon in a strange position. It would be resolved if we just

> discontinue this addon and have people use the TurboWarp button addon instead

and considering that all those suggestions about adding Interpolation, HQ Pen, etc. are being rejected, this would be fair enough, and it would abide by the DRY principle. The problem is (as always) that previous users will be confused, and people who design projects for 60FPS might want to enable the feature in the editor, which is impossible with TurboWarp mode.